setup.py

```diff
@@ -13,6 +13,9 @@
     scripts=["voc"],
     packages=["volvooncall"],
     install_requires=list(open("requirements.txt").read().strip().split("\n")),
```

Comment:
It's usually not recommended for libraries to load the requirements directly from a requirements.txt file. Both serve different purposes which aren't necessarily compatible to each other.

**install_requires** should offer wide dependency ranges so the library can be installed together with other packages without creating dependency conflicts.
**requirements.txt** should specify a concrete version for each dependency used for testing, usually the latest one.

My suggestion, add the `aiohttp<4.0` dependency here directly and create (only) one `requirements.txt` file which can / should pin specific versions.

Better yet, instead of `setup.py`, define `install_requires` and later `extras_require` in `setup.cfg`.
https://setuptools.pypa.io/en/latest/userguide/dependency_management.html